### DISCUSSION TITLE
Website Fonts

### DISCUSSION THREAD

**Discussion by WirthmU** [aaronwmorris/indi-allsky#1426] [General]
I am having strange graphic effects on the website picture.
This is from the SQM tab:

![image](https://github.com/user-attachments/assets/ba350d20-0b3f-48de-9e42-24f45e809299)

In the settings I couldn't find how to correct blurry fonts on the picture and black triangles around SQM values below. I'm using Google Chrome browser.

**Comment by aaronwmorris** [comment]
Yeah, this happens on everyones' systems.  Basically, the text is drawn twice on the HTML canvas, first for the black ouline, second the gray foreground, to ensure readability on any background.

The background text has the spikes.  I noticed it when I first programmed it, but I never went back to investigate further.

I found this:  https://stackoverflow.com/questions/19988099/how-to-prevent-ugly-spikes-in-canvas-font-rendering

I will work on dealing the spikes.

**Comment by aaronwmorris** [comment]
Merged #1427 to fix the text background.

**Reply by WirthmU** [reply]
It's fixed now - thank you

![image](https://github.com/user-attachments/assets/e1ab8ed8-a35a-49d1-bdd6-4d8171df569e)
